airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json

```diff
@@ -68,11 +68,17 @@
         "title": "Password",
         "order": 6
       },
+      "jdbc_url_params": {
+        "description": "Additional properties to pass to the jdbc url string when connecting to the database formatted as 'key=value' pairs separated by the symbol '&'. (example: key1=value1&key2=value2&key3=value3).",
```

Comment:
nit 
```suggestion
        "description": "Additional properties to pass to the JDBC URL string when connecting to the database formatted as 'key=value' pairs separated by the symbol '&'. (example: key1=value1&key2=value2&key3=value3).",
```

Reply:
I took this from here: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/source-java-jdbc/src/main/resources/spec.json.hbs#L41

I can make this change, but would you like me to also then just change this capitalization everywhere?

Reply:
Let's keep it as it is.

Reply:
Hey @misteryeo @alafanechere said the same thing here and I replied, would you prefer me to just update the capitalization everywhere?